$MirOS: ports/lang/egcs/gcc4.4/patches/patch-gcc_common_opt,v 1.1 2009/12/05 15:38:53 tg Exp $

	• add -Werror-maybe-reset
	• stub out -Wbounded, -fhonour-copts for mgcc compat
	• enable SSP by default like mgcc

--- gcc/common.opt.orig	Sat Mar 28 17:28:21 2009
+++ gcc/common.opt	Thu May 29 22:27:02 2014
@@ -82,6 +82,10 @@ Wattributes
 Common Var(warn_attributes) Init(1) Warning
 Warn about inappropriate attribute usage
 
+Wbounded
+Common
+MirBSD mgcc compatibility (ignored)
+
 Wcast-align
 Common Var(warn_cast_align) Warning
 Warn about pointer casts which increase alignment
@@ -102,6 +106,10 @@ Werror=
 Common Joined
 Treat specified warning as error
 
+Werror-maybe-reset
+Common
+If environment variable GCC_NO_WERROR is set, act as -Wno-error
+
 Wextra
 Common Warning
 Print extra (possibly unwanted) warnings
@@ -573,6 +581,10 @@ fguess-branch-probability
 Common Report Var(flag_guess_branch_prob) Optimization
 Enable guessing of branch probabilities
 
+fhonour-copts
+Common RejectNegative
+MirBSD mgcc compatibility (ignored)
+
 ; Nonzero means ignore `#ident' directives.  0 means handle them.
 ; Generate position-independent code for executables if possible
 ; On SVR4 targets, it also controls whether or not to emit a
@@ -1089,7 +1101,7 @@ Common RejectNegative Joined
 -fstack-limit-symbol=<name>	Trap if the stack goes past symbol <name>
 
 fstack-protector
-Common Report Var(flag_stack_protect, 1)
+Common Report Var(flag_stack_protect, 1) Init(1)
 Use propolice as a stack protection method
 
 fstack-protector-all
